Hippo.Web/ClientApp/src/components/Account/RequestForm.tsx

```diff
@@ -66,7 +72,7 @@ export const RequestForm = () => {
           <span className="status-color">{context.user.detail.firstName}</span>
         </h3>
         <p>
-          You don't seem to have an account on Farm yet. If you'd like access,
+          You don't seem to have an account on the {cluster} cluster yet. If you'd like access,
```

Comment:
Worthwhile getting the cluster description here?

Reply:
i didn't think it was worth it.  but if we put it in the context then maybe..